### PR TITLE
feat: Add metrics in EOTS manager

### PR DESCRIPTION
**Pull request by gusin13**
Fixes - https://github.com/babylonchain/finality-provider/issues/57

This pr succeeds https://github.com/babylonchain/finality-provider/pull/265 and adds pending metrics related to EOTS manager. 

All the metrics mentioned in https://babylon-chain.atlassian.net/wiki/spaces/BABYLON/pages/203980929/Finality+Provider+Metrics have been added now.

### CODE REVIEW

**Review by gitferry — APPROVED** [review]
Lgtm! One minor comment:

File: metrics/eots_collectors.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	return eotsMetricsInstance
+}
+
+// IncrementEotsCreatedKeysCounter increments the EOTS created keys counter
+func (em *EotsMetrics) IncrementEotsCreatedKeysCounter() {
+	em.EotsCreatedKeysCounter.Inc()
+}
+
+// IncrementEotsFpTotalGeneratedRandomnessCounter increments the EOTS signature counter
+func (em *EotsMetrics) IncrementEotsFpTotalGeneratedRandomnessCounter(fpBtcPkHex string) {
```
[the remark this answers is not in the document]
Comment (gusin13) [reply metrics/eots_collectors.go:84]: so we are incrementing only by 1 at a time so thats why we used Inc(), Add() would be used if we are adding a bigger number to the counter no?

**Comment by gusin13** [comment]
cc @filippos47 , metric related ports introduced in this pr, i will make a new pr on helm/k8s later to accomodate.

File: metrics/eots_collectors.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (gusin13) [reply metrics/eots_collectors.go:84]: note - i am merging this for now, will be happy to change anything later if any concerns

**Pull request merged**